recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ocp" %}
+{% set version = "7.5.2beta" %}
```

Comment:
beta stuff seems to happen more than infrequently with some of the packages in this ecosystem... is a real release forthcoming?

Reply:
Yes